alphagov/gsp-ci-pipelines
# Add Notary into the mix

## What

We're adding harbor into the mix, which will require some credentials
for both authenticating with as well as performing notary actions for
any images.

RoleBinding is a required resource to allow particular team accessing
the secrets placed in corresponding K8s namespace.

The secrets should be exposed from up top. The secret needs to live in
the same namespace as the CI system.

## How to review

- Code review
- Apply pipeline to the cluster with harbor
- See it succeed
